Procfile

```diff
@@ -1 +1,4 @@
-web: gunicorn --pythonpath backend app:app
\ No newline at end of file
+web: gunicorn --pythonpath backend app:app
+
+web: bokeh serve --port=$PORT --allow-websocket-origin=social-good-hackathon.herokuapp.com --address=0.0.0.0 --use-xheaders app.py
```

Comment:
Since the parent folder is outside backend/ and frontend/ files need to be specified with backend/ or frontend/ prefixes.

Please change app.py to backend/main.py for the Bokeh to run successfully